examples/functions/golang/backfill/mmf/matchfunction.go

```diff
@@ -254,7 +254,7 @@ func newMatch(num int, profile string, tickets []*pb.Ticket, b *pb.Backfill) pb.
 	t := time.Now().Format("2006-01-02T15:04:05.00")
 
 	return pb.Match{
-		MatchId:       fmt.Sprintf("profile-%s-time-%s-num-%d", matchName, t, num),
+		MatchId:       fmt.Sprintf("profile-%s-time-%s-num-%d", profile, t, num),
```

Comment:
Instead of profile you could also use an ID like we use for tickets.

Reply:
As I understand from xid code, they use proximity as Unix() seconds and add global counter. So for multiple MMF pods we can have same xids generated.